daml-lf/archive/proto_breaking_changes.sh

```diff
@@ -11,17 +11,9 @@ readonly config_file="${arch_dir}/buf.yaml"
 readonly stable_dir="${arch_dir}/src/stable/protobuf"
 readonly main_dir="${arch_dir}/src/main/protobuf"
 
-# We check that the stable directories contain exactly 2 proto files
-for dir in ${stable_dir}; do
+# We check the directories contains exactly 2 proto files
+for dir in "${stable_dir}" "${main_dir}"; do
    find "${dir}/" -follow -name '*.proto' | wc -l | grep -x 2
 done
 
-# We check that the main directory contains exactly 3 proto files
-find "${main_dir}/" -follow -name '*.proto' | wc -l | grep -x 3
-
-# This is kind of broken, it only checks daml_lf_(1|2).proto for wire compatibility between main and
-# stable. The daml_lf.proto files are ignored because main/.../daml_lf.proto and
-# sable/.../daml_lf.proto declare different proto packages. This is not too bad because
-# daml_lf.proto is mostly empty while daml_lf_(1|2).proto contains most of the definitions, but it
-# is not ideal.
```

Comment:
I would keep this comment since it is still true

Reply:
right, there's no more `daml_lf_1.proto`, but the comment is still right about the `_2` protos.
> The daml_lf.proto files are ignored because main/.../daml_lf.proto and stable/.../daml_lf.proto declare different proto packages

`daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf.proto` has `package daml_lf_dev;`
`daml-lf/archive/src/stable/protobuf/com/daml/daml_lf_2_1/daml_lf.proto` has `package daml_lf_2_1;`

To be honest I don't fully understand everything here but I remember that @paulbrauner-da found this, he might know better